src/test/java/at/medunigraz/imi/bst/TestTopic.java

```diff
@@ -0,0 +1,55 @@
+package at.medunigraz.imi.bst;
+
+import de.julielab.ir.model.QueryDescription;
+import de.julielab.ir.model.QueryDescriptionAttribute;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class TestTopic extends QueryDescription {
+    @QueryDescriptionAttribute
+    private String query;
+    @QueryDescriptionAttribute
+    private List<String> stopFilteredTermList;
+    @QueryDescriptionAttribute
+    private String[] stopFilteredTermArray;
+    @QueryDescriptionAttribute
+    private double weight;
+    @QueryDescriptionAttribute
+    private List<String[]> friends;
+
+    public TestTopic withFriends(String[]... friends) {
```

Comment:
Friends?

Reply:
Sure. Everyone should have some! Why not also our TestTopic ;-)